pygromos/data/simulation_parameters_templates/emin.imd

```diff
@@ -24,7 +24,7 @@ FORCE
 # 	 BONDS 	 ANGLES 	 IMPROPER 	 DIHEDRAL 	 ELECTROSTATIC 	 VDW
  	 1 	 1 	 1 	 1 	 1 	 1 	 
 # 	 NEGR 	 NRE
- 	 4 	 71 	 72 	 73 	 2863 	 
+ 	 3 	 1 	 2 	 3 	 
```

Comment:
are these changes required or is it template beautifying? :)  I think in theory the functions should automatically adapt these values. (If not I need to check why they don't :) )

Reply:
i did not intentionally change this - can you just keep the original version?

Reply:
jap I can do that :)